main.py

```diff
@@ -477,6 +477,33 @@ def list(self, dpid=None):
 
         return jsonify(switch_flows)
 
+    @rest("v2/stored_flows")
+    @rest("v2/stored_flows/<dpid>")
```

Comment:
@gretelliz great intention trying to have something similar to `v2/flows`, but considering how the [flow documents were modelled](https://github.com/kytos-ng/flow_manager/blob/master/db/models/__init__.py#L115-L121) and MongoDB query operators, for `dpid` we can also support more flexibility here, including a list of dpids. With that said, I think it'd be simpler to only have `v2/stored_flows` in terms of maintainability, and then also turn `dpids` as an URL query param similar to `state` that you've implemented, that way dpids could be parsed as a list of dpids separated by comma or something similar. That way you don't also have to try to filter for the existing dpids, if `dpids` weren't specified you won't include any dpid filter and return all documents, simplifying the code here too (in the previous implementation that was strategic since it was filtering in in-memory stored values)